src/components/BackupButton.tsx

```diff
@@ -21,17 +21,17 @@ export default function BackupButton ({
   const backupEvents = new EventTarget()
   const space = selectedSpace ?? storacha?.spaces?.[0]
   async function onClick () {
-    if (space && bluesky.userProfile && bluesky.agent && storacha.client) {
+    if (space && (bluesky.userProfile || bluesky.session) && bluesky.agent && storacha.client) {
```

Comment:
and then we should be able to avoid this `|| bluesky.session` stuff

Reply:
this won't be necessary anymore if we're not using the custom `auth/callback` route anymore, right?... and `bluesky.userProfile` will be available.

Reply:
yep exactly!

Reply:
wow! i really shouldn't have gone the *"manual"* oauth callback route. Thank you for catching this. I'll make a cleanup commit and push now.